src/components/editor/InputListLOC.jsx

```diff
@@ -41,7 +41,7 @@ class InputListLOC extends Component {
     this.props.handleSelectedChange(payload)
   }
 
-  hasPropertyRemark(propertyTemplate) {
+  hasPropertyRemark = (propertyTemplate) => {
```

Comment:
Parens not needed in this context, but they don't break anything and we're already using both styles throughout this codebase, so no biggie here.

Reply:
I personally like the parens, but if we all wanted to agree on a style and make it so throughout the code, I would stick to it.

Reply:
Agreed. Ideally we would lean on eslint to do this for us, but I'm loath to suggest we make nitpicky style changes at this point.